python/ray/tune/suggest/__init__.py

```diff
@@ -72,6 +75,7 @@ def _import_sigopt_search():
         return SigOptSearch
 
     SEARCH_ALG_IMPORT = {
+        "variant_generator": _import_variant_generator,
```

Comment:
Should we add aliases here? Like `tune` or `random`. It is random search (and grid search) after all.

Reply:
Thanks for the suggestion! I personally think a `tune` key might be a little confusing since I personally haven't seen that used to indicate a random/grid search. I definitely agree with adding `random`. @krfricke what do you think about adding `random` and `grid`?

Reply:
Let's add `random` then. I wouldn't do `grid` as we still require using `tune.grid_search` so it could be confusing for people who instantiate a grid shim but don't define a grid search parameter.